File: ProEventos-em-Vue/src/models/Evento.ts
import { Lote } from "./Lote"
import { Palestrante_Evento } from "./Palestrante_Evento"
import { RedeSocial } from "./RedeSocial"

export interface Evento {
    id: number
    local: string
    dataEvento: Date
    tema: string
    qtdPessoas: number
    imagemURL: string
    telefone: string
    email: string
    lotes: Array<Lote>
    redeSociais: Array<RedeSocial>
    palestrantesEventos: Array<Palestrante_Evento>
}

File: Back/src/ProEventos.Domain/Entities/Evento.cs
using System;
using System.Collections.Generic;

namespace ProEventos.Domain.Entities
{
    public class Evento : BaseEntity
    {
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }
        public string Tema { get; set; }
        public int QtdPessoas { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public List<Lote> Lotes { get; set; }
        public List<RedeSocial> RedeSociais { get; set; }
        public List<Palestrante_Evento> PalestrantesEventos { get; set; }
    }
}